chain/consensus/mir/config.go

```diff
@@ -79,20 +95,27 @@ func NewConfig(
 		MembershipSourceValue: membershipSource,
 	}
 
-	if maxBlockDelay <= 0 {
-		maxBlockDelay = DefaultMaxBlockDelay
+	var maxBlockDelaySec time.Duration
+	if maxBlockDelay > 0 {
+		maxBlockDelaySec = time.Duration(maxBlockDelay) * time.Second
+	} else {
+		maxBlockDelaySec = DefaultMaxBlockDelay
+		maxBlockDelay = 1
```

Comment:
I'm quite confused by this. If the `maxBlockDelay` argument was directly of type `time.Duration`, this gymnastics would not be necessary.
But even if there is a reason for `maxBlockDelay` to be an `int` (in which case I would call that `maxBlockDelaySec`), I still think this could probably be made simpler.

Reply:
Fixed